models/timm/beit_base_patch16_224.py

```diff
@@ -0,0 +1,19 @@
+# labels: name::beit_base_patch16_224 author::timm
+import torch
+import timm
+from mlagility.parser import parse
+
+# Parsing command-line arguments
+batch_size = parse(["batch_size"])
+
+# Creating model and set it to evaluation mode
+model = timm.create_model("beit_base_patch16_224", pretrained = False)
+model.eval()
+
+# Creating inputs
+input_size = model.default_cfg["input_size"]
+batched_input_size = tuple(batch_size) + input_size
```

Comment:
This doesn't look right to me, shouldn't it be `batched_input_size = tuple(batch_size) * input_size`?

Reply:
I see your confusion. We are simply appending one dimension to the `input_size`. Here is one example from one of the models:

```
>>> batch_size=[1]
>>> input_size=(3,244,244)
>>> tuple(batch_size) + input_size
(1, 3, 244, 244)
```

Reply:
WTF what a weird property of python 😂 

Is there some tuple.append() or something that makes it more obvious what is going on? Or do I just need to git gud?

Reply:
This is weird. Tuples are immutable.

Reply:
yeap. That is just how Python works. There's no append() or extend() method for tuples. The + operator seems to be the recommended way of concatenating tuples.